autopilot/graph.go

```diff
@@ -82,7 +82,7 @@ func (d dbNode) Addrs() []net.Addr {
 //
 // NOTE: Part of the autopilot.Node interface.
 func (d dbNode) ForEachChannel(cb func(ChannelEdge) error) error {
-	return d.node.ForEachChannel(d.tx, func(tx *bolt.Tx,
+	return d.node.ForEachChannel(nil, func(tx *bolt.Tx,
```

Comment:
If it's closed or not? The idea is to not require a new db transaction for each node we explore, and instead do it under a single database transaction.

Reply:
Was able to avoid this change by no longer keeping the `Node` objects around.